runtime/doc/treesitter.txt

```diff
@@ -490,6 +490,12 @@ still highlighted the same as other operators: >query
 
     "!=" @operator (#set! conceal "≠")
 <
+To conceal an entire line (next unconcealed line is drawn in it's stead), a
```

Comment:
```suggestion
To conceal an entire line, a
```
I think that's clear enough in this context.

Reply:
IDK; in this context (without the knowledge of "vertical conceal"), "conceal an entire line" still reads as "conceal the entire line horizontally" to me. Should we refer to `nvim_buf_set_extmark()`'s properties above then?